Apps/Examples/Examples/All Examples/ShowHideLayerExample.swift

```diff
@@ -0,0 +1,164 @@
+        }
+    }
+
+    @objc func toggleContourLayerVisibility(sender: UISwitch) {
+        let style = mapView.mapboxMap.style
+        // Update the contour layer's visibility based on whether the switch
+        // is on.
+        do {
+            try style.updateLayer(withId: contourLayerId) { (layer: inout CircleLayer) in
+                layer.visibility = .constant(sender.isOn ? .visible : .none)
+            }
+        } catch {
+            print("Failed to update the visibility for layer with id \(contourLayerId). Error: \(error.localizedDescription)")
+        }
+    }
+
+    func addVisibilitySwitches() {
+        // Create switches to toggle the layers' visibility.
+        let museumSwitch = UISwitch()
+        museumSwitch.addTarget(self, action: #selector(toggleMuseumLayerVisibility(sender:)), for: .valueChanged)
+        museumSwitch.isOn = true
+        museumSwitch.translatesAutoresizingMaskIntoConstraints = false
+        view.insertSubview(museumSwitch, aboveSubview: mapView)
+
+        let contourSwitch = UISwitch()
+        contourSwitch.addTarget(self, action: #selector(toggleContourLayerVisibility(sender:)), for: .valueChanged)
+        contourSwitch.isOn = true
+        contourSwitch.translatesAutoresizingMaskIntoConstraints = false
+        view.insertSubview(contourSwitch, aboveSubview: mapView)
+
+        // Add labels for the toggles.
+        let museumLabel = UILabel()
+        museumLabel.text = "Show museums"
+        museumLabel.adjustsFontSizeToFitWidth = true
+        museumLabel.backgroundColor = .white
+        museumLabel.translatesAutoresizingMaskIntoConstraints = false
+        view.insertSubview(museumLabel, aboveSubview: mapView)
+
+        let contourLabel = UILabel()
+        contourLabel.text = "Show contours"
+        contourLabel.adjustsFontSizeToFitWidth = true
+        contourLabel.backgroundColor = .white
+        contourLabel.translatesAutoresizingMaskIntoConstraints = false
+        view.insertSubview(contourLabel, aboveSubview: mapView)
+
+        // Layout the switches and labels.
+        NSLayoutConstraint.activate([
+            museumSwitch.topAnchor.constraint(equalTo: mapView.safeAreaLayoutGuide.topAnchor, constant: 20),
+            museumSwitch.leadingAnchor.constraint(equalTo: mapView.safeAreaLayoutGuide.leadingAnchor, constant: 20),
+            museumLabel.widthAnchor.constraint(equalToConstant: 100),
+            museumLabel.leadingAnchor.constraint(equalTo: museumSwitch.trailingAnchor, constant: 10),
+            museumLabel.heightAnchor.constraint(equalTo: museumSwitch.heightAnchor),
+            museumLabel.centerYAnchor.constraint(equalTo: museumSwitch.centerYAnchor),
+            contourSwitch.topAnchor.constraint(equalTo: museumSwitch.bottomAnchor, constant: 20),
+            contourSwitch.centerXAnchor.constraint(equalTo: museumSwitch.centerXAnchor),
```

Comment:
Keeping layouts of sibling views as decoupled as possible is typically good for performance and for maintenance. In this case, keep the vertical spacing between the switches, but replace this center x constraint with a leading constraint relative to the view